### DISCUSSION TITLE
使用get 请求 getDefinitionsProductType 下的 Schema 链接 失败

### DISCUSSION THREAD

**Discussion by zjjzhengjunjie** [amzn/selling-partner-api-models#3383] [General]
javax.net.ssl.SSLPeerUnverifiedException: Certificate for <selling-partner-definitions-prod-iad.s3.amazonaws.com> doesn't match any of the subject alternative names: [*.s3.amazonaws.com, s3.amazonaws.com]

**Comment by zjjzhengjunjie** [comment]
之前请求是可以的，忽然就不行了，是需要升级什么吗？

**Comment by chapmanjw** [comment]
@zjjzhengjunjie We are not seeing this issue when downloading the schemas in our tests. It sounds like there is an issue on the client side. Are you able to view those links in your web browser?
